mobile/src/screens/LoginScreen2.tsx

```diff
@@ -0,0 +1,69 @@
+import { useNavigation } from "@react-navigation/native";
+import { StackNavigationProp } from "@react-navigation/stack";
+import React from "react";
+import { Text, View, Image, Pressable, TextInput, SafeAreaView } from "react-native";
+
+import mascotImg from "../../assets/STEMy_Mascot.png";
+import {AppStylesheet} from '../styles/AppStylesheet'
+
+import { RootStackParamList } from "./RootStackParams";
+
+
+
+
+type logIn2ScreenProp = StackNavigationProp<RootStackParamList, "Login page 2">;
+
+const LoginScreen2 = () => {
+  const navigation = useNavigation<logIn2ScreenProp>();
+  const [usernameText, changedUsernameText] = React.useState("Example Padding");
+  const [passwordText, changedPasswordText] = React.useState("Example Padding");
```

Comment:
Typically when using the useState hook we follow the format of `const [value, setValue] = useState("value")` where value is the variable holding the value, setValue is the function for assigning it, and the parameter to useState is the default value. You can see more about this here https://reactjs.org/docs/hooks-state.html. We should follow this naming syntax here and whenever we use useState. Also we can import useState directly so we don't need to do `React.useState` by importing it as `import React, { useState } from 'react';`

Reply:
done